.github/workflows/ci-release.yaml

```diff
@@ -0,0 +1,175 @@
+name: Release CI
+
+on:
+  workflow_dispatch:
+    inputs:
+      version-overwrite:
+        required: false
+        default: ''
+        description: 'Use this to overwrite the version number to release, otherwise uses the current SNAPSHOT version (expected format x.y.z)'
+        type: string
+
+jobs:
+  prepare-release:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.varaibles.outputs.version }}
+      next-version: ${{ steps.varaibles.outputs.next-version }}
+      release-branch: ${{ steps.varaibles.outputs.release-branch }}
```

Comment:
`varaibles` looks like a typo here?

Reply:
:sweat_smile: yes that is in fact a typo, however I seem to have been efficient and made it across the entire workflow, which means it still works at least.

I shall fix it though, thanks for catching that :+1: